src/coreclr/jit/importer.cpp

```diff
@@ -17153,12 +17316,21 @@ bool Compiler::impReturnInstruction(int prefixFlags, OPCODE& opcode)
             impBashVarAddrsToI(op2);
             op2 = impImplicitIorI4Cast(op2, info.compRetType);
             op2 = impImplicitR4orR8Cast(op2, info.compRetType);
-            // Note that we allow TYP_I_IMPL<->TYP_BYREF transformation, but only TYP_I_IMPL<-TYP_REF.
+// Note that we allow TYP_I_IMPL<->TYP_BYREF transformation, but only TYP_I_IMPL<-TYP_REF.
+#ifdef TARGET_LOONGARCH64
```

Comment:
Please remove this. There should be no architecture-specific code in the importer.

Reply:
had removed.
Thanks